.travis.yml

```diff
@@ -0,0 +1,35 @@
+language: generic
+
+os:
+  - linux
+  - osx
+
+env:
+  matrix:
+  - CONDA_PY=27
+  - CONDA_PY=35
+  - CONDA_PY=36
```

Comment:
The package is noarch, so we only want to build packages for one of these, right?

Reply:
How would we test the library for regressions against 2.7 while still building a package once? I don't the best practice for this, so any input would be appreciated.